test/unit/app/util-test.js

```diff
@@ -2,8 +2,8 @@ import { strict as assert } from 'assert'
 import {
   getEnvironmentType,
   sufficientBalance,
-  isPrefixedFormattedHexString,
 } from '../../../app/scripts/lib/util'
+import { isPrefixedFormattedHexString } from '../../../shared/modules/utils'
```

Comment:
Nit: These tests should be moved to a different file :sweat_smile:

Reply:
😅